apps/files/src/components/FileSharingSidebar.vue

```diff
@@ -1,13 +1,206 @@
 <template>
-    <div>
-      <p>file sharing sidebar</p>
+  <div>
+    <div v-if="selectedFiles.length > 1">
+      <span v-translate>Please choose only a single File</span>
+    </div>
+    <div v-else>
+     <oc-spinner v-if="sharesLoading"></oc-spinner>
+    <oc-accordion v-if="!sharesLoading" uk-accordion="collapsible: false; multiple: true">
```

Comment:
If I collapse one item I can't collapse the other one. Is it on purpose like this? If `collapsible: false;` is removed then the problem is gone.

Reply:
no idea if intended .... feel free to change an push to this branch